openmldb-import run error

**Bug Description**
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/curator/connection/ConnectionHandlingPolicy
        at org.apache.curator.framework.CuratorFrameworkFactory.builder(CuratorFrameworkFactory.java:82)
        at org.apache.curator.framework.CuratorFrameworkFactory.newClient(CuratorFrameworkFactory.java:108)
        at org.apache.curator.framework.CuratorFrameworkFactory.newClient(CuratorFrameworkFactory.java:94)
        at com._4paradigm.openmldb.importer.Importer.getTableMetaData(Importer.java:266)
        at com._4paradigm.openmldb.importer.Importer.Load(Importer.java:151)
        at com._4paradigm.openmldb.importer.Importer.main(Importer.java:331)
Caused by: java.lang.ClassNotFoundException: org.apache.curator.connection.ConnectionHandlingPolicy
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
        ... 6 more


**Expected Behavior**



**Steps to Reproduce**

1. 
2. 
3. 
4.

--- a/docs/zh/tutorial/data_import.md
+++ b/docs/zh/tutorial/data_import.md
@@ -11,8 +11,10 @@
 
 ```bash
 > cd java/openmldb-import
-> mvn package 
+> mvn package
 ```
+
+`mvn package -Dmaven.test.skip=true` 跳过测试。
 
 ## 2. 导数工具使用
 
@@ -21,7 +23,7 @@
 --help可以展示出所有的配置项，星号表示必填项。
 
 ```bash
-> java -jar openmldb-import.jar --help
+> java -jar openmldb-import-1.0-SNAPSHOT.jar --help
 ```
 
 ```
@@ -32,8 +34,8 @@ Usage: Data Importer [-fhV] [--create_ddl=<createDDL>] --db=<dbName>
                      [,<files>...] [--files=<files>[,<files>...]]...
 insert/bulk load data(csv) to openmldb
       --create_ddl=<createDDL>
-                            if force_recreate_table is true, provide the create
-                              table sql
+                            if table is not exists or force_recreate_table is
+                              true, provide the create table sql
 *     --db=<dbName>         openmldb database
   -f, --force_recreate_table
                             if true, we will drop the table first
@@ -60,12 +62,13 @@ insert/bulk load data(csv) to openmldb
 
 重要配置的项目说明：
 
-- `--db=<dbName>`: 库名。库名可以是不存在的，importer可以帮助创建。
-- `--table=<tableName>`: 表名。表名可以是不存在的，importer可以帮助创建。但请注意，如果导入到已存在的表，需要表内数据为空，否则将会极大影响导入效率。
-- `--files=<files>[,<files>...]`: 导入源文件。目前源文件只支持csv格式的本地文件，并且csv文件必须有header，文件的列名和表的列名必须一致，顺序可以不一样。
-- `--zk_root_path=<zkRootPath>`和`--zk_cluster=<zkCluster>`: 集群版OpenMLDB的ZK地址和路径
-
 - `--importer_mode=<mode>`: 导入模式，支持insert和bulkload两种方式。默认配置为bulkload.
+
+- `--zk_cluster=<zkCluster>`和`--zk_root_path=<zkRootPath>`: 集群版OpenMLDB的ZK地址和路径。
+- `--db=<dbName>`: 库名。库名可以是不存在的，importer可以帮助创建。
+- `--table=<tableName>`: 表名。表名可以是不存在的，importer可以帮助创建，需配置`--create_ddl`。但请注意，如果导入到已存在的表，需要表内数据为空，否则将会极大影响导入效率。
+
+- `--files=<files>[,<files>...]`: 导入源文件。目前源文件只支持csv格式的本地文件，并且csv文件必须有header，文件的列名和表的列名必须一致，顺序可以不一样。
 
 ## 3. 大规模的数据导入
 

--- a/java/openmldb-import/pom.xml
+++ b/java/openmldb-import/pom.xml
@@ -33,20 +33,22 @@
             <version>4.13.1</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.4paradigm.openmldb</groupId>
+            <artifactId>openmldb-common</artifactId>
+            <version>0.6.0</version>
+        </dependency>
         <dependency>
           <groupId>com.4paradigm.openmldb</groupId>
           <artifactId>openmldb-jdbc</artifactId>
-          <version>0.2.0</version>
+          <version>0.6.0</version>
         </dependency>
+
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
             <version>3.16.1</version>
-        </dependency>
-        <dependency>
-            <groupId>com.4paradigm.openmldb</groupId>
-            <artifactId>openmldb-common</artifactId>
-            <version>0.3.2</version>
         </dependency>
         <dependency>
             <groupId>com.baidu</groupId>
@@ -62,9 +64,8 @@
         <!-- https://mvnrepository.com/artifact/org.apache.curator/apache-curator -->
         <dependency>
             <groupId>org.apache.curator</groupId>
-            <artifactId>apache-curator</artifactId>
-            <version>5.1.0</version>
-            <type>pom</type>
+            <artifactId>curator-client</artifactId>
+            <version>4.2.0</version>
         </dependency>
 
         <!--日志开始-->
@@ -235,7 +236,6 @@
     </dependencies>
 
     <build>
-        <finalName>openmldb-import</finalName>
         <resources>
             <resource>
                 <directory>src/main/resources</directory>

--- a/java/openmldb-import/src/main/java/com/_4paradigm/openmldb/importer/Importer.java
+++ b/java/openmldb-import/src/main/java/com/_4paradigm/openmldb/importer/Importer.java
@@ -74,7 +74,7 @@ public class Importer {
     @CommandLine.Option(names = "--table", description = "openmldb table", required = true)
     private String tableName;
 
-    @CommandLine.Option(names = "--create_ddl", description = "if force_recreate_table is true, provide the create table sql", defaultValue = "")
+    @CommandLine.Option(names = "--create_ddl", description = "if table is not exists or force_recreate_table is true, provide the create table sql", defaultValue = "")
     private String createDDL;
     @CommandLine.Option(names = {"-f", "--force_recreate_table"}, description = "if true, we will drop the table first")
     private boolean forceRecreateTable;

